Tests/DataPipeline/DataPipelineCompatibilityTests.swift

```diff
@@ -146,12 +146,12 @@ class DataPipelineCompatibilityTests: IntegrationTest {
         XCTAssertFalse(savedEvent.containsKey("platform"))
 
         var expectedData = givenDefaultAttributes
+        expectedData["cio_sdk_version"] = diGraphShared.sdkClient.sdkVersion
```

Comment:
Because `sdkVersion` is now from removed `DeviceInfo` and added to `SdkClient` instead